server/client.go

```diff
@@ -917,13 +920,21 @@ func (c *client) flushOutbound() bool {
 		}
 	}
 
+	// Check that if there is still data to send and writeLoop is in wait,
+	// we need to signal
+	if c.out.pb > 0 && c.out.sgw {
```

Comment:
Why not just signal, why do we need the bool? I think we may just want to signal on partial?